symfony/client/src/orangehrmPerformancePlugin/components/ReviewersAutoComplete.vue

```diff
@@ -0,0 +1,77 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <employee-autocomplete
+    label="Reviewers"
+    :multiple="true"
+    :clear="false"
+    :create-options="loadEmployees"
+    required
+  />
+</template>
+
+<script>
+import EmployeeAutocomplete from '@/core/components/inputs/EmployeeAutocomplete';
+import {APIService} from '@/core/util/services/api.service';
+
+export default {
+  name: 'ReviewersAutoComplete',
```

Comment:
I think the purpose of adding this component is being able to exclude certain employees from showing. please add the prop that will take the excluded Ids. [refer here](https://github.com/orangehrm/orangehrm/blob/5.0/symfony/client/src/orangehrmTimePlugin/components/ProjectAutocomplete.vue#L44)

Reply:
addded to send the request to get both current and past emps

Reply:
> addded to send the request to get both current and past emps

Is that the outcome you need here?

Reply:
So right now I'm only trying to get both current and past employees as reviewers using the multiple autocomplete for employees

Reply:
I remember you wanted to exclude certain employees from showing in autocomplete? I thought you added this component to address that issue as well as the multiple selecting. if not we can directly use employee autocomplete with passing `multiple="true"`

Reply:
Yeah sure I'll try to add that part as well and commit